packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3225,6 +3228,14 @@ export class ContainerRuntime
 				},
 				error,
 			);
+
+			const testDelayOverrideMs =
```

Comment:
If the error thrown contains a `retryAfter` field, that should be honored. If it's more than 10 seconds, this delay won't help very much.

Reply:
So this delay will fire after the retryAfter when we try to summarize a second time. In the `trySummarize` method we do wait for the delay seconds. If we receive an unknown ack, we will wait 10 seconds to restart as well.

Reply:
Oh wait, I totally missed that this is in container runtime. This code is especially important in `SummaryGenerator` where summary fails. It's not need here because summaries didn't fail.

Reply:
Also, with restart there is no second attempt. There is just the one attempt.

Reply:
In this scenario, we don't know if summarizer failed. It's probably best to make sure we do not burn summarizer cycles in the worst case scenario. Waiting 10 seconds should not be an issue unless a large number of ops occurs in a very short timespan, which we should have a mitigation for when virtual sequencing rolls out.